spec/lib/board_spec.rb

```diff
@@ -3,15 +3,65 @@
 require_relative '../../lib/board'
 
 RSpec.describe 'Board' do
+  let(:board) { Board.new }
+
   context 'when Board#board is called' do
     it 'should return an array of arrays with three nils in each' do
-      board_instance = Board.new
+      expect(board.board).to eq [
+        [nil, nil, nil],
+        [nil, nil, nil],
+        [nil, nil, nil]
+      ]
+    end
+  end
+
+  context 'when position_to_row_and_col' do
+    it 'should return an array with row and col given 1 for position' do
+      row_col = board.position_to_row_and_col 1
+      row = row_col[0]
+      col = row_col[1]
+
+      expect(row).to equal(0)
+      expect(col).to equal(0)
+    end
 
-      # Allows access to private instance variable
-      board = board_instance.instance_variable_get(:@board)
+    it 'should return an array with row and col given 8 for position' do
+      row_col = board.position_to_row_and_col 8
```

Comment:
Is this method part of the public interface of `board`? Would a client of this class want to use this? I think your initial intuition to make this method private was correct. I remind myself often that we want to test behavior and not implementation. Can we exercise the code by testing the public method `make_move` and by only providing input and looking at the output ensure that our behavior is correct without having to make implementation details like this method public?